esmvaltool/cmorizers/data/formatters/datasets/esacci_cloud.py

```diff
@@ -0,0 +1,404 @@
+
+    if short_name in ["clt", "ctp"]:
+        cube = utils.fix_coords(cube)
+        # Fix metadata and  update version information
+        attrs = copy.deepcopy(cfg["attributes"])
+        attrs["mip"] = var["mip"]
+        utils.set_global_atts(cube, attrs)
+        # Save variable
+        utils.save_variable(
+            cube, short_name, out_dir, attrs, unlimited_dimensions=["time"]
+        )
+
+    # Fix metadata and  update version information
+    cube_day = utils.fix_coords(cube_day)
+    attrs_day = copy.deepcopy(cfg["attributes"])
+    attrs_day["mip"] = var["mip"]
+    attrs_day["version"] += "-daylight"
+    utils.set_global_atts(cube_day, attrs_day)
+    # Save variable
+    utils.save_variable(
+        cube_day,
+        short_name,
+        out_dir,
+        attrs_day,
+        unlimited_dimensions=["time"],
+    )
+
+
+def _concatenate_and_save_monthly_cubes(
+    short_name, var, cubes, out_dir, attach, cfg, cmor_info
+):
+    """Concatinate monthly files and save."""
+    # After gathering all cubes for all years, concatenate them
+    cube = cubes.concatenate_cube()
+
+    # # Check for missing months
+    # year0 = cube.coord("time").cell(0).point.year
+    # year1 = cube.coord("time").cell(-1).point.year
+    # logger.debug(year0, year1)
+    # cube = _check_for_missing_dates(year0, year1, cube, cubes)
+
+    if attach == "-AMPM":
+        cube = monthly_statistics(cube)
+
+    # Regrid the cube to the target grid (e.g., 0.5x0.5)
+    cube = regrid(cube, target_grid="0.5x0.5", scheme="area_weighted")
+
+    # Fix units and handle any special cases like 'clt'
+    if short_name == "clt":
+        cube.data = 100 * cube.core_data()  # Example conversion
+    else:
+        if "raw_units" in var:
+            cube.units = var["raw_units"]
+        cube.convert_units(cmor_info.units)
+
+    # Set global attributes and fix metadata
+    attrs = copy.deepcopy(cfg["attributes"])
+    attrs["mip"] = var["mip"]
+    attrs["version"] += attach
+    utils.set_global_atts(cube, attrs)
+
+    # Save the processed variable
+    utils.save_variable(
+        cube, short_name, out_dir, attrs, unlimited_dimensions=["time"]
+    )
+
+
+def _process_daily_file(
+    ifile, inum, short_name, var, cmor_info, cubes, cubes_day
+):
+    """Extract variable from daily file."""
+    logger.info("CMORizing file %s", ifile)
+    # Extract raw names from the variable dictionary
+    raw_var = var.get("raw", short_name)
+
+    for ivar, raw_name in enumerate(raw_var):
+        logger.info("Extracting raw variable %s", raw_name)
+
+        # Define variable for daylight
+        if "_asc" in raw_name:
+            illum = "illum_asc"
+        else:
+            illum = "illum_desc"
+
+        # Load cube using a constraint based on the raw_name
+        daily_cube = iris.load_cube(ifile, NameConstraint(var_name=raw_name))
+        daily_cube_ilum = iris.load_cube(ifile, NameConstraint(var_name=illum))
+
+        # Set arbitrary time of day
```

Comment:
Could you add a comment why it's okay to set an arbitrary time?

Reply:
Done.